### ISSUE TITLE
Applications of Library Carpentry in "real" life

### ISSUE THREAD

**Issue by vitsie** [LibraryCarpentry/week-two-library-carpentry#3]
Hi, I am interested to get an insight into how the participants in Library Carpentry think they will apply what we're learning in their working lives. As session 2 went on, I could see more applications for working through large collections of data. What are you inspired to use your new skills to do at work? Are you already using them?

**Comment by drjwbaker** [comment]
Thanks for kicking this off Kathryn!

**Comment by HKollef** [comment]
I can already see how to apply this to any research work, whether that be for my masters or as part of a future position. The ability to parse so much data from multiple documents at the same time seems incredibly useful, especially when paired with something like a Twitter data set or multiple excel sheets, etc.  

Edited to add: I will be using this for a research paper that I'm writing, so I'm already finding it useful!

**Comment by rosiehigman** [comment]
I'm hoping to use it for pulling out data from multiple spreadsheets on e-books relevant to the departments that I support but getting the data into a workable state is proving challenging!

**Comment by drjwbaker** [comment]
@rosiehigman: getting data into a workable state is always a challenge. Week Four will discuss Open Refine, a tool I rely on a huge amount for this very purpose. Hopefully this will give you a place to start.

**Comment by GentlemanSykes** [comment]
Certinaly for me the attraction is to begin working with larger sets of data, particularly data for informing collection management, so circulation stats. Particularly I;d like to combine/compare stats on physical books and ebooks in a less manual way. Getting to that data and getting to understand it will be in itself a challenge, so what we've learnt so far is certainly inspiring me to believe that this is worth doing. I;ve yet to put this into practice as its still a busy time at work and at home but its certinaly firing off connections in my mind and I hope to build on this maybe small scale to begin with to get my confidence.

**Comment by ClareP** [comment]
Yes, for me it's been pretty inspiring to see what can be done with our data, I just worry that, especially in terms of bibliographic data the issue might be trying to get the stuff out of our LMS in the first place in order to work on it. I know I can export datasets from ours (Alma) but creating meaningful sets in the first place could be a stumbling block. Certainly I can envisage ways that the data in our institutional repository could be cleaned up (we have issues with duplicates of journal names, 'and' vs '&' etc) and although I haven't tried it for myself I can see that EPrints, which we use for our IR, allows us to export data in loads of different formats so yay for them! :+1:

**Comment by drjwbaker** [comment]
@GentlemanSykes: If I were a writer of movie posters I'd be emblazoning everything that touches Library Carpentry with 'inspiring' right now...  

@ClareP: 'I know I can export datasets from ours (Alma) but creating meaningful sets in the first place could be a stumbling block'. What do you mean by 'sets' here?

**Comment by ClareP** [comment]
Sorry being jargony! I mean like a batch of marc records or .mrc files. Eg I'd like to have a list of isbns maybe in a spreadsheet or something that I can feed into alma so I can match marc records that contain them then export the .mrc files as a set to work on outside of the LMS.

**Comment by drjwbaker** [comment]
Library jargon. I miss it... Looks like you have to figure out:

1) what data you want to work with outside of the LMS
2) how to work with that data once you have it outside of the LMS
3) how to interface with the LMS to get the data you want (if not, back to 1) based on how you can interface with the LMS)

**Comment by ClareP** [comment]
Right and from what I have learnt so far at library carpentry it seems that once we get the data out of the wretched LMS the world is our oyster in terms of what we can do with it...

**Comment by ostephens** [comment]
@ClareP one option might be to export more than you need then filter down outside the LMS?

**Comment by drjwbaker** [comment]
Otherwise known as the nuclear option.

**Comment by cjclib** [comment]
The nuclear option is surely pressing the "delete all records" big red button in the LMS? ;)

I'm sure there will be ways to get what you need (or as Owen said, a bigger set that you can pare down) out of Alma, I'll bet there are people doing that kind of thing regularly Clare. Just need to figure out who to ask (often half the battle)

**Comment by ClareP** [comment]
Yeah I'm sure there's something that can be done - have only worked with alma for a few months so will hopefully uncover some more tricks. Owen that sounds like a reasonable way of doing it too :)

**Comment by drjwbaker** [comment]
Collated responses to the question last night on how you and your fellow attendees might use what they've learnt up the mid-point in real life. I hope there are some points here that fire off further ideas! https://github.com/LibraryCarpentry/week-two-library-carpentry/blob/master/LibCarp-mid-term-feedback-anon.txt
